facturacion_mexico/facturacion_fiscal/tests/test_nota_credito_descuento_relacion_01.py

```
@@ -0,0 +1,530 @@
+	apply_descuento_to_lines,
+	build_descuento_description,
+	credit_note_lines_use_discount_account,
+	get_cuenta_descuentos,
+)
+
+CUENTA_DESC = "501-005 - Descuentos y bonificaciones - TC"
+
+UUID_ORIGEN_C = "550E8400-E29B-41D4-A716-446655440000"
+
+
+def _make_ffm(motivo, uuid=UUID_ORIGEN_C):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_make_ffm`

(ANN202)